sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -55,6 +85,14 @@ def load(
     :paramtype trim_prefixes: Optional[List[str]]
     :keyword key_vault_options: Options for resolving Key Vault references
     :paramtype key_vault_options: ~azure.appconfiguration.provider.AzureAppConfigurationKeyVaultOptions
+    :keyword refresh_all: One or more settings whos modification will trigger a full refresh after a fixed interval.
+     This should be a list of Key-Label pairs for specific settings (filters and wildcards are not supported).
+    :paramtype refresh_all: List[Tuple[str, str]]
+    :keyword refresh_only: One or more specific settings that will be refreshed after a fixed interval.
```

Comment:
Would `keep_fresh` be a better name?

Reply:
Definitely open to finding better names for these two parameters....
I rather like `keep_fresh` - it specifically implies that this should be a list of settings present within the provider.